starter-code/styles/modules.css

```diff
@@ -9,10 +9,32 @@
   padding: 0 0 .2em 1em;
 }
 
+input {
+    height: 2.5rem;
+    width: 2.5rem;
+    position: absolute;
+    right: 1rem;
+    top: 1rem;
+    opacity: 0;
+}
+
+#menuToggle input:checked ~ ul {
+  opacity: 1;
+}
+
 .icon-menu {
+ font-size: 2.5rem;
+ /*margin: 1rem;*/
+ position: absolute;
+ top: 1rem;
+ right: 1rem;
+}
+
+.icon-menu:active {
   /* TODO: Add styles for the icon-menu class (whether it be the
       hamburger font or an image of your choosing). It should be
       bigger, and off to the right. */
+      font-size: 34px;
```

Comment:
this will be the stay the same size as the rest of the view adjusts in size,
is there a relative unit that will change with the screen size?